src/pages/AddLibraryPage.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <q-page class="full-height column q-pa-md bg-grey-1">
+    <h4
+      class="q-ma-none q-mt-md q-mb-lg q-mr-auto"
+      data-cy="page_add_library_title"
+    >
+      {{ $t('AddLibraryPage.text.title') }}
+    </h4>
+    <q-form
+      class="column"
+      style="flex: 1;"
```

Comment:
put in `<style>` or use a quasar class if possible

Reply:
done